### PR TITLE
fix: Added Blur to placeholder with base64

### PR DESCRIPTION
**Pull request by MichaelUnkey**
## What does this PR do?

Fixes # (issue)

*If there is not an issue for this, please create one first. This is used to tracking purposes and also helps use understand why this PR exists*

## Type of change

* [x] Bug fix (non-breaking change which fixes an issue)
* [ ] Chore (refactoring code, technical debt, workflow improvements)
* [ ] Enhancement (small improvements)
* [ ] New feature (non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
* [ ] This change requires a documentation update

## How should this be tested?

* Test A
* Test B

### Required

* [ ] Filled out the "How to test" section in this PR
* [ ] Read [Contributing Guide](./CONTRIBUTING.md)
* [ ] Self-reviewed my own code
* [ ] Commented on my code in hard-to-understand areas
* [ ] Ran `pnpm build`
* [ ] Ran `pnpm fmt`
* [ ] Checked for warnings, there are none
* [ ] Removed all `console.logs`
* [ ] Merged the latest changes from main onto my branch with `git pull origin main`
* [ ] My changes don't cause any responsiveness issues

### Appreciated

* [ ] If a UI change was made: Added a screen recording or screenshots to this PR
* [ ] Updated the Unkey Docs if changes were necessary

<sub><a href="https://huly.app/guest/unkey?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJsaW5rSWQiOiI2Njc5ZDI0NjJhNmFiOWVlZjU1Nzc5ZmIiLCJndWVzdCI6InRydWUiLCJlbWFpbCI6IiNndWVzdEBoYy5lbmdpbmVlcmluZyIsIndvcmtzcGFjZSI6InctamFtZXMtdW5rZXktNjY2YWYwNzktNjliNzJiMzYyNi05ZGM2MGUiLCJwcm9kdWN0SWQiOiIifQ.Qmb6w0B8ep-5iFseevYfgrnzPcihU8BDeIwZqfBltw4">Huly&reg;: <b>ENG-1792</b></a></sub>

### CODE REVIEW

**Comment by chronark** [comment]
I think this is too easy to forget and too cumbersome to do for every new image we add
can't we add this in a central place instead and then use that new component?

File: apps/www/components/image-with-blur.tsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import Image, { type ImageProps } from "next/image";
+import React from "react";
+
+type ImageWithBlurProps = ImageProps;
+
+export default function ImageWithBlur(props: ImageWithBlurProps) {
```
Comment (chronark) [inline apps/www/components/image-with-blur.tsx:6]: no default exports please
```suggestion
export function ImageWithBlur(props: ImageWithBlurProps) {
```

**Review by chronark — CHANGES_REQUESTED** [review]

File: apps/www/components/image-with-blur.tsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import Image, { type ImageProps } from "next/image";
+import React from "react";
+
+type ImageWithBlurProps = ImageProps;
```
Comment (chronark) [inline apps/www/components/image-with-blur.tsx:4]: what value does this type add?

File: apps/www/components/ip-whitelisting-bento.tsx [hunk 3]
```diff
@@ -1,10 +1,10 @@
-import Image from "next/image";
 import ip from "../images/ip.svg";
+import ImageWithBlur from "./image-with-blur";
```
Comment (chronark) [inline apps/www/components/ip-whitelisting-bento.tsx:2]: how are you doing default imports if the component is not default exported?

Comment (MichaelUnkey) [reply apps/www/components/ip-whitelisting-bento.tsx:2]: Good question

**Comment by chronark** [comment]
@perkinsjr @MichaelUnkey what's the status here?

**Comment by perkinsjr** [comment]
If Mike has provided the requested changes from you, and it build correctly. 

Then it can be merged.

**Review by perkinsjr — APPROVED** [review]

**Review by perkinsjr — APPROVED** [review]

**Pull request merged**